Modules/Microsoft365DSC/DSCResources/MSFT_IntuneRoleAssignment/MSFT_IntuneRoleAssignment.schema.mof

```diff
@@ -0,0 +1,20 @@
+[ClassVersion("1.0.0.0"), FriendlyName("IntuneRoleAssignment")]
+class MSFT_IntuneRoleAssignment : OMI_BaseResource
+{
+    [Write, Description("The unique idenfier for an entity. Read-only.")] String Id;
+    [Write, Description("Description of the Role Assignment.")] String Description;
+    [Write, Description("The display or friendly name of the role Assignment.")] String DisplayName;
+    [Write, Description("List of ids of role scope member security groups. These are IDs from Azure Active Directory.")] String ResourceScopes[];
+    [Write, Description("List of DisplayName of role scope member security groups. These are Displayname from Azure Active Directory.")] String ResourceScopesDisplayNames[];
+    [Write, Description("The list of ids of role member security groups. These are IDs from Azure Active Directory.")] String Members[];
+    [Write, Description("The list of Displaynames of role member security groups. These are Displaynamnes from Azure Active Directory.")] String MembersDisplayNames[];
+    [Write, Description("The Role Definition Id.")] String RoleDefinition;
+    [Write, Description("The Role Definition Displayname.")] String RoleDefinitionDisplayName;
+    [Write, Description("Present ensures the policy exists, absent ensures it is removed."), ValueMap{"Present","Absent"}, Values{"Present","Absent"}] string Ensure;
```

Comment:
Need to update the definition to reflect that this resource deals with Roles and not policies.